rollup.config.js

```diff
@@ -28,8 +28,7 @@ export default [
       FileUploadButton: "src/button/file-upload/FileUploadButton.tsx",
       Spinner: "src/spinner/Spinner.tsx",
       Countdown: "src/countdown/Countdown.tsx",
-      ProgressBar: "src/progress-bar/ProgressBar.tsx",
-      Textarea: "src/form/textarea/Textarea.tsx"
```

Comment:
Was this a mistake? Why are we removing `Textarea` ?

Reply:
Ops! I supposed that there is duplicated `Textarea` but there wasn't. My bad, sorry! 😅 

Actually, we can provide finding eslint issues while npm is running. Should we add a script to `package.json` for this?

Reply:
Yes, I think we should add a step to build command for linting. This way, we can make sure there are no errors before publishing to npm.
I think we can create an issue (or update, if you opened one for the issue Bilge mentioned below), and add it later.